pkg/cmd/create/plan/network/fetchers/nutanix/fetcher.go

```
@@ -0,0 +1,111 @@
+			}
+		}
+		sourceNetworks = append(sourceNetworks, sourceNetwork)
+	}
+
+	klog.V(4).Infof("Nutanix network fetcher - found %d source networks", len(sourceNetworks))
+	return sourceNetworks, nil
+}
+
+// FetchTargetNetworks is not supported; Nutanix is a source provider only.
+func (f *NetworkFetcher) FetchTargetNetworks(ctx context.Context, configFlags *genericclioptions.ConfigFlags, providerName, namespace, inventoryURL string, insecureSkipTLS bool) ([]forkliftv1beta1.DestinationNetwork, error) {
+	return nil, fmt.Errorf("Nutanix provider does not support target network fetching - only OpenShift is supported as migration target")
```

golangci-lint (2.12.2)

[error] ST1005: error strings should not be capitalized

(staticcheck)